src/plugin/scoring-metadata/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why a pom.xml? Nutch is still built using ant. It shouldn't be included as it only confuses users which then try to run "mvn" and wonder why the compilation fails or the packages are not bundled properly.